code/src/compenents/IntroPage.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+
+const IntroPage = ({name, onNameChange, onStepChange}) => {
+  
+    return (
+      <div>
+        <article className="letter-container" aria-label="Letter container">
+          <form className="form" onSubmit={event => event.preventDefault()}>
+            <h1 tabindex="0">Welcome to Hogwarts!</h1>
+            <p tabindex="0">Before stepping on the Hogwarts express at Platform 9 3/4, we need to get some information about you so you will have the best 7 years at Hogwarts school of witchcraft and wizardry!</p>
+            <h2 tabindex="0">Let's start with your name:</h2>
```

Comment:
Cool that you keep the same background and change the text. I will remember that!